.travis.yml

```diff
@@ -15,13 +15,32 @@
 # specific language governing permissions and limitations
 # under the License.
 
-dist: trusty
-language: java
-jdk:
-  - oraclejdk8
+language: generic
+
+jobs:
+  include:
+    - name: "Linux amd64"
+      arch: amd64
+      dist: bionic
+    - name: "Linux aarch64"
+      arch: arm64-graviton2
+      dist: focal
+      virt: lxd
+      group: edge
+
+cache:
+  directories:
+    - $HOME/.m2
+
 before_install:
   - export MAVEN_SKIP_RC="true" # Travis has settings in /etc/mavenrc. We want to override them. See https://github.com/travis-ci/travis-ci/issues/4613
-install:
-  - # Skip mvn install. See https://docs.travis-ci.com/user/languages/java/
+  - sudo apt update
+  - sudo apt install -y openjdk-8-jdk
```

Comment:
Manually install OpenJDK 8 because it is not available on TravisCI ARM64 nodes. The oldest supported one is JDK 11